locales/en-US/app.ftl

```diff
@@ -752,6 +756,38 @@ SourceView--no-known-cors-url =
 SourceView--network-error-when-obtaining-source =
     There was a network error when fetching the URL { $url }: { $networkErrorMessage }
 
+# Displayed below SourceView--cannot-obtain-source, if the browser was queried
+# for source code using the symbolication API, and this query returned an error.
+# Variables:
+#   $apiErrorMessage (String) - The raw internal error message from the API, not localized
+SourceView--browser-api-error-when-obtaining-source =
+    The browser's symbolication API returned an error: { $apiErrorMessage }
```

Comment:
```suggestion
    The browser’s symbolication API returned an error: { $apiErrorMessage }
```

nit: use curly apostrophes (also in other strings)

Reply:
Oh right, I keep forgetting!

It would be great to have a automatic lint for this.